qiskit/circuit/library/arithmetic/adder/ripple_carry_adder.py

```diff
@@ -0,0 +1,116 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Compute the sum of two qubit registers using ripple-carry approach."""
+
+from qiskit.circuit import QuantumCircuit, QuantumRegister, AncillaRegister
+
+
+class RippleCarryAdder(QuantumCircuit):
+    r"""A ripple-carry circuit to perform addition on two qubit registers.
```

Comment:
Could you mention here that it's an inplace addition?

Reply:
Mentioned that it's in-place, thanks!

Reply:
I think everything is clear here now, so I will mark the conversation as resolved for the time being :)